en/core-api/credit-card.md

```diff
@@ -1,57 +1,87 @@
+
+| Attribute | Value | Note |
+|-----------|-------|-------|
+| `data-environment`| sandbox; production | Enter the values depending on the environment. |
+| `data-client-key`| Client key | For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys). |
+
+For more details about the API, refer to [Get Token](https://api-docs.midtrans.com/#get-token).
+
+#### Endpoints
+
+| Method | URL                                       |
+| ------ | ----------------------------------------- |
+| GET    | https://api.sandbox.midtrans.com/v2/token |
+
+#### **Query Parameters**
 
-| Attribute | Value |
-|-----------|-------|
-| `data-environment`| Input `sandbox` or `production` (API environment)|
-| `data-client-key`| Input **client key** [by following previous section.](/en/midtrans-account/overview.md#retrieving-api-access-keys) |
+| Parameter      | Description                   | Type   | Notes                                                        |
+| -------------- | ----------------------------- | ------ | ------------------------------------------------------------ |
+| client_key     | Your Client Key.              | String | For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys). |
+| card_number    | The card number.              | String | The card number is 16 digit number in XXXX XXXX XXXX XXXX format. |
+| card_exp_month | The expiry month on the card. | String | The expiry month is in  MM format.                           |
+| card_exp_year  | The expiry year on the card.  | String | The expiry year is in YY format.                             |
+| card_cvv       | The CVV number on the card.   | String | The CVV number is in XXX format.                             |
 
-Link: [*More detailed definition*](https://api-docs.midtrans.com/#get-token)
+#### Headers
 
-### Get Card Token JS Implementation
-To retrieve card `token_id`, we will be using `MidtransNew3ds.getCardToken` function. Implement the following Javascript on our payment page.
+| Header Name   | Description                                            | Required | Values           |
+| ------------- | ------------------------------------------------------ | -------- | ---------------- |
+| Accept        | The format of the data to be returned.                 | Required | application/json |
+| Content-Type  | The format of the data to be posted.                   | Required | application/json |
+| Authorization | The authentication method used to access the resource. | Required | Basic Og==       |
+
+#### Sample Request
+```bash
+curl --location --request GET 'https://api.sandbox.midtrans.com/v2/token?client_key={Your client Key}&card_number=4811%201111%201111%201114&card_exp_month=12&card_exp_year=24&card_cvv=123' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Basic Og=='
+```
```

Comment:
Why is this API request via CURL for get-token still being explained here? We have already discussed to not show the CURL method, if you want to show it put it on advanced-feature page please.

Reply:
Sorry. It was a mistake from my side. I forgot to remove it.

Reply:
ok thanks

Reply:
This one doesn't seem to be resolved yet, I can still see the API request & Curl example still being displayed. Please fix. thx

Reply:
This one is not yet resolved @rashmikampurath

Reply:
Sorry, I have done the changes and committed the file.